index.html

```diff
@@ -2,17 +2,16 @@
 <html lang="en">
 
 <head>
-  <meta charset="UTF-8">
   <title>Space Ripples Reveal Big Bang’s Smoking Gun</title>
   <link rel="stylesheet" href="./style.css">
   <meta name="viewport" content="width=device-width, initial-scale=1.0" property=''>
+  <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
 </head>
 
```

Comment:
## Remove unnecessary spacing
- [ ] Kindly remove the spacing in between the heading of the main article and the main image.
- [ ] While at it. I welcome you to centre the social icons and style them to appear as they appear on the original page. See inspiration below.

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)[|[Clone Page](https://raw.githack.com/amiraabouhadid/project1/page-styling/index.html)|
|:--:|:--:|
|![top-image](https://user-images.githubusercontent.com/22345961/108412887-651d7b80-723b-11eb-8e90-690788aa47c8.png)|![top-image-clone](https://user-images.githubusercontent.com/22345961/108412903-69499900-723b-11eb-9bf7-9646141400ec.png)|